src/main/java/com/kk/strong/model/dto/WorkoutSessionDto.java

```diff
@@ -0,0 +1,18 @@
+package com.kk.strong.model.dto;
+
+import com.kk.strong.model.WorkoutCategory;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class WorkoutSessionDto {
+
+    private Long id;
+    private Long userId;
+    private LocalDateTime timestamp;
+    private WorkoutCategory workoutCategory;
+    private List<ExerciseDto> exercises = new ArrayList<>();
```

Comment:
btw what is the reason behind `= new ArrayList<>();` I didn't see that often

Reply:
It's actually not necesarry in this class - so I'll remove it. But in the case of the user class, we have a list of roles and when creating the user we want to add a default role to his roles, so it's useful that the empty list is initialized when the user is created